src/noc/containers/noc-check-and-submit/noc-check-and-submit.component.spec.ts

```diff
@@ -147,6 +147,27 @@ describe('NocCheckAndSubmitComponent', () => {
     expect(spyOnDispatchToStore).toHaveBeenCalledWith(new fromFeature.SetAffirmationError(affirmationError));
   });
 
+  it('should set notify error', () => {
+    component.affirmationAgreed = true;
+    component.notifyEveryParty = false;
+    const affirmationError = {
+      AFFIRMATION_NOTIFY_EVERY_PARTY_ERROR
+    };
+    component.verifyAndSubmitNoC();
+    expect(spyOnDispatchToStore).toHaveBeenCalledWith(new fromFeature.SetAffirmationError(affirmationError));
+  });
+
+  it('should set affirmation error', () => {
```

Comment:
please change to:
"should set affirmation agreed error when user doesn't confirm their answers are accurate"